WordPress/gutenberg
# Update some icons to address aliasing on high-res screens

Part of #https://github.com/WordPress/gutenberg/issues/49311, focusing on common blocks/UI icons first. Updates a few icons to remove the aliasing that occurs on high-resolution screens. 

## Why?
We want our icons to look crisp and present well.

## How?
Updating icons within the icons package. This does not change the spirit of the icons, just adjusts the icon paths a little to not use `0.25` X and Y values. 

## Testing Instructions
Check the icons in the following locations: 

1. Heading block. 
2. Gallery block.
3. Grid block.
4. Paragraph block.
5. Row block.
6. Stacked block.
7. List view icon.

## Screenshots or screencast 
<img width="628" alt="CleanShot 2023-03-27 at 16 30 43" src="https://user-images.githubusercontent.com/1813435/228059392-de7b215d-3741-4415-bcce-bd9f9f32d0fd.png">
<img width="628" alt="CleanShot 2023-03-27 at 16 31 11" src="https://user-images.githubusercontent.com/1813435/228059493-c235c54b-1901-45d9-994f-1d1d687eec54.png">
